### DISCUSSION TITLE
Next Market

### DISCUSSION THREAD

**Discussion by thetaspere** [thetaspere/theta#28] [Polls]
Lets do the vote!
We will focus on this market

Closed at : `15 Oct 2022`

**Comment by Aw3n** [comment]
The current price of Txbit is $0.000498978492 with a 24-hour trading volume of $27,029.22. The price has risen by 3.7% in the last 24 hours. It has a outstanding offer of 0 TXBIT coins and a maximum offer of 2 billion. Currently, the most active market on which this cryptocurrency trades is Txbit. What was the highest Txbit price? Txbit reached an all-time high of $0.022584123659 on Dec 02, 2021 (10 months). What was the lowest price of Txbit? Txbit hit its all-time low of $0.000212211114 on Aug 10, 2022 (about 2 months). What was the trading volume of Txbit over 24 hours? Txbit's 24-hour trading volume is $27,029.22. Why avoid the inevitable, vote txbit be smart!

**Reply by thetaspere** [reply]
hahaha
